naga/src/front/glsl/error.rs

```diff
@@ -16,7 +16,13 @@ use crate::SourceLocation;
 use crate::{proc::ConstantEvaluatorError, Span};
 
 #[cfg(feature = "termcolor")]
-use codespan_reporting::term::termcolor::WriteColor;
+use codespan_reporting::term::termcolor::WriteColor as ErrorWrite;
+
+#[cfg(all(not(feature = "termcolor"), feature = "stderr"))]
+use std::io::Write as ErrorWrite;
+
+#[cfg(not(any(feature = "termcolor", feature = "stderr")))]
+use core::fmt::Write as ErrorWrite;
```

Comment:
suggestion: Let's move this to `naga::Error`,  if we can. It'd be good to keep the "fiddly `cfg` bits" in the same place. 🙂 I like the simplification in use sites!